ipykernel/kernelapp.py

```diff
@@ -16,17 +16,16 @@
 from zmq.eventloop.zmqstream import ZMQStream
 
 from IPython.core.application import (
-    BaseIPythonApplication, base_flags, base_aliases, catch_config_error
+        BaseIPythonApplication, base_flags, base_aliases, catch_config_error
 )
 from IPython.core.profiledir import ProfileDir
 from IPython.core.shellapp import (
-    InteractiveShellApp, shell_flags, shell_aliases
+        InteractiveShellApp, shell_flags, shell_aliases
 )
 from IPython.utils import io
 from ipython_genutils.path import filefind, ensure_dir_exists
-from traitlets import (
-    Any, Instance, Dict, Unicode, Integer, Bool, DottedObjectName, Type,
-)
+from traitlets import (Any, Instance, Dict, Unicode, Integer, Bool,
+                       DottedObjectName, Type, default)
```

Comment:
I won't ask you to revert these changes, but FWIW, I prefer the form of all of these before this PR to after.